src/utilities/util_verify.c

```diff
@@ -60,61 +60,65 @@ int
 util_verify(WT_SESSION *session, int argc, char *argv[])
 {
     WT_CURSOR *cursor;
+    WT_DECL_ITEM(config);
     WT_DECL_RET;
-    size_t size;
+    WT_SESSION_IMPL *session_impl;
     int ch;
-    char *config, *dump_offsets, *key, *uri;
-    bool abort_on_error, do_not_clear_txn_id, dump_address, dump_all_data, dump_key_data,
-      dump_blocks, dump_layout, dump_tree_shape, dump_pages, read_corrupt, stable_timestamp, strict;
+    char *dump_offsets, *key, *uri;
+    bool abort_on_error, dump_all_data, dump_key_data;
 
-    abort_on_error = do_not_clear_txn_id = dump_address = dump_all_data = dump_key_data =
-      dump_blocks = dump_layout = dump_tree_shape = dump_pages = read_corrupt = stable_timestamp =
-        strict = false;
-    config = dump_offsets = uri = NULL;
+    abort_on_error = dump_all_data = dump_key_data = false;
+    dump_offsets = uri = NULL;
+    session_impl = (WT_SESSION_IMPL *)session;
+
+    WT_RET(__wt_scr_alloc(session_impl, 0, &config));
 
     while ((ch = __wt_getopt(progname, argc, argv, "acd:kSstu?")) != EOF)
         switch (ch) {
         case 'a':
             abort_on_error = true;
             break;
         case 'c':
-            read_corrupt = true;
+            WT_ERR(__wt_buf_catfmt(session_impl, config, "read_corrupt,"));
             break;
         case 'd':
             if (strcmp(__wt_optarg, "dump_address") == 0)
-                dump_address = true;
+                WT_ERR(__wt_buf_catfmt(session_impl, config, "dump_address,"));
             else if (strcmp(__wt_optarg, "dump_blocks") == 0)
-                dump_blocks = true;
+                WT_ERR(__wt_buf_catfmt(session_impl, config, "dump_blocks,"));
             else if (strcmp(__wt_optarg, "dump_layout") == 0)
-                dump_layout = true;
+                WT_ERR(__wt_buf_catfmt(session_impl, config, "dump_layout,"));
             else if (strcmp(__wt_optarg, "dump_tree_shape") == 0)
-                dump_tree_shape = true;
+                WT_ERR(__wt_buf_catfmt(session_impl, config, "dump_tree_shape,"));
             else if (WT_PREFIX_MATCH(__wt_optarg, "dump_offsets=")) {
                 if (dump_offsets != NULL) {
                     fprintf(
                       stderr, "%s: only a single 'dump_offsets' argument supported\n", progname);
                     return (usage());
                 }
                 dump_offsets = __wt_optarg + strlen("dump_offsets=");
+                WT_ERR(__wt_buf_catfmt(session_impl, config, "dump_blocks=[%s],", dump_offsets));
```

Comment:
This line is a bit confusing to me - why is `dump_offsets` handled in this way? I had a quick read of the [util verify docs](https://source.wiredtiger.com/develop/struct_w_t___s_e_s_s_i_o_n.html#a0334da4c85fe8af4197c9a7de27467d3) but its still a bit unclear.

Reply:
If you mean that the `dump_offsets` option is inserted as `dump_blocks` then this is indeed a bug :) Fixed in the latest commit, thanks for noticing!